cfme/tests/infrastructure/test_host.py

```diff
@@ -162,3 +180,22 @@ def test_tag_host_after_provider_delete(provider, appliance, setup_provider, req
     provider.wait_for_delete()
     tag = host.add_tag()
     request.addfinalizer(lambda: host.remove_tag(tag))
+
+
+def test_250_vmware_hosts_loading(appliance, create_250_hosts):
+    """
+    Test to automate BZ1580569
+    """
+    # Without the patch, this will cause the process to consume roughly 10+ Gigs of RAM
+    # due to a poorly optimized database query
+    rails_console = appliance.ssh_client.run_rails_console(
+        "MiqReport.load_from_view_options(Host, User.where(:userid => 'admin').first)"
+        ".paged_view_search", timeout=60
+    )
+    assert rails_console.success
+
+    # Check it loads in the UI fast
+    host_collection = appliance.collections.hosts
+    view = host_collection.appliance.browser.create_view(HostsView)
```

Comment:
**Optional** I think you can combine 198 and 199 since `host_collection` is not used anywhere else

Reply:
Alternatively, why use create_view() here and not a call to `navigate_to()`? I don't see where the navigation would occur, explicitly, anywhere otherwise (in the fixture)